guides/centring_accuracy.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: guides
+title: Guides | Centring Accuracy 
+description: This guide teaches you how to centre the accuracy on the playfield.
+---
+
+# Centring Accuracy
+
+This guide will teach you how to centre the accuracy counter on the playfield.
+
+Note that by doing this, the percentage symbol after the accuracy will disappear. This is due to osu! stopping to load the texture and loading only the hitbox, due to its size. Also note that this will work in all game modes, but will look off in osu!taiko as the accuracy counter is shifted downwards due to the taiko scoring animation.

```

Comment:
but didnt mania also get completly messed up by this?

Reply:
nope, works fine in mania remember?
only combo and judgements are centered on the stage